docs/codespaces.md

```diff
@@ -0,0 +1,33 @@
+# Developing in GitHub Codespaces
+
+Navigate to your branch in Github, select 'Code' and create a new codespace. This will create a virtual machine specific to the branch, running Postgres, Redis etc.
+
+## Test data
+
+If you want a reasonable set of test data, enter the following command in the terminal: `bundle exec rake setup_local_dev_data`
+
+## Viewing the support console
+
+To view the support console, you'll need to have a SupportUser set up. If you've already run `setup_local_dev_data`, this has already happened. If not, open a Rails console (see below) and run `SupportUser.create( dfe_sign_in_uid: 'dev-support', email_address: 'support@example.com')`
+
+## Making your codespace visible to other team members
+
+- Go to the Ports tab
+- The website will be running on port 3000
+- The context menu for the port will give you the option to adjust 'port visibility'. "Private to Organization" is probably the right option here
+
+## Running tests
+
+- Open a new terminal (`+` button)
+- Run `bundle exec rspec spec/path-to-tests`
+
+## Modifying data in a Rails console
+
+- Open a new terminal (`+` button)
+- Run `bundle exec rails console`
+- Now you can do things like `ApplicationForm.first.update(first_name: 'Falafel')`
+
+## Gotchas
+
+- The codespace will send emails with application URLs relative to http://localhost:3000. Thus for example when you receive a magic link, you'll need to copy the URL and change the host to whatever Codespaces has generated for you.
```

Comment:
```suggestion
- The codespace will send emails with application URLs relative to http://localhost:3000. For example, when you receive a magic link, you'll need to copy the URL and change the host to whatever Codespaces has generated for you.
```

Reply:
correct my slightly pompous use of the word 'thus' = insult my whole family